docs/source/whatsnew/0.3.4.txt

```diff
@@ -30,3 +30,12 @@ Bug Fixes
   a numpy array to a DataFrame (:issue:`273`).
 * Fix appending into a sql table from chunks not returning the table.
   (:issue:`278`).
+
+Miscellaneous
+-------------
+
+* Adds :func:``~odo.utils.copydoc`` function to copy docstrings from one object
```

Comment:
I don't mind putting in the extra work for better docs. How do I build the docs locally to test the changes?